services/ingestion/etl/inforegio/xml/test/unit/lib/__snapshots__/transform.spec.js.snap

```diff
@@ -46,9 +46,16 @@ Object {
     Object {
       "address": "",
       "centroid": null,
-      "country_code": "EL",
+      "country_code": "GR",
       "location": null,
-      "nuts2": "EL0",
+      "nuts": Array [
+        Object {
+          "code": "EL0",
```

Comment:
`EL0` doesn't exist 😬 `EL3`, `EL4`, `EL5` and `EL6` do. By the way, the level should be `1`, not `2`